vscode-extension/src/config.ts

```diff
@@ -0,0 +1,15 @@
+import {ConfigurationScope, workspace} from 'vscode';
+
+export type Config = {
+  pathToRelay: string | null;
+  outputLevel: string;
+};
+
+export function getConfig(scope?: ConfigurationScope): Config {
+  const configuration = workspace.getConfiguration('relay', scope);
+
+  return {
+    pathToRelay: configuration.get('pathToRelay') ?? null,
```

Comment:
Just curious: Does the fact that we declared `null` as the default in our `package.json` mean that VSCode will provide us with that default value when we call `.get` here? Or is that simply a documentation description, which we must actually implement ourselves here?

Reply:
I assume anyone can open their `settings.json` and put whatever they want. Let me test this out to see what the behavior is and how much safety we actually need to put in.